styles.css/main.css

```diff
@@ -187,9 +185,10 @@ h1 {
   font-size: 25px;
   font-weight: normal;
   margin-top: 5px;
-  color:whitesmoke;
+  color:rgba(0,0,0,0);
   background: rgb(2,0,36);
-background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(22,148,209,1) 23%, rgba(0,212,255,1) 100%);
+  background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(22,148,209,1) 23%, rgba(0,212,255,1) 100%);
+  background-clip: text;
```

Comment:
Set your text color to transparent (alpha 0 in color) and then use background-clip to have the background clip to the text